src/initialization/InitElement.cpp

```diff
@@ -529,7 +529,7 @@ namespace detail
             amrex::ParticleReal ds = 0.0;
             pp_element.queryAdd("ds", ds);
 
-            elements::mixin::LinearTransport::Map6x6 transport_map = elements::mixin::LinearTransport::Map6x6::Identity();
+            Map6x6 transport_map = Map6x6::Identity();
```

Comment:
`Map6x6` is currently not declared anywhere

Reply:
Maybe `LinearMap::Map6x6`?

Reply:
This is from another PR, will revert here.